gherkin-utils/java/README.md

```diff
@@ -0,0 +1,7 @@
+[![Maven Central](https://img.shields.io/maven-central/v/io.cucumber/gherkin.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22io.cucumber%22%20AND%20a:%22gherkin%22)
```

Comment:
Please fix the full contents of this file (replace gherkin with gherkin-utils). Fix the content.